modules/instrument_builder/jsx/react.app.js

```diff
@@ -0,0 +1,485 @@
+TabPane = React.createClass({
```

Comment:
The indentation in this file also seems weird.. why are you using 8 spaces instead of 4? Most of our code uses 4 spaces for indentation.